hedera-dependency-versions/build.gradle.kts

```diff
@@ -103,7 +104,7 @@ dependencies.constraints {
         api(gav("org.hyperledger.besu.secp256k1", besuNativeVersion))
         api(gav("org.json", "20230227"))
         api(gav("org.junit.jupiter.api", "5.9.0"))
-        api(gav("org.junit.platform.engine", "1.9.3"))
+        api(gav("org.junit.platform.engine", "1.9.1"))
```

Comment:
To be consistent with the version packaged in Gradle. Otherwise the dependency analysis can get confused and the are potentially issues if running on module path (two versions of "engine" jar on the module path). This is a known issue in Gradle that should be solved in Gradle 8.x